ton_client/client/src/tests/mod.rs

```diff
@@ -319,40 +342,51 @@ impl TestClient {
             .unwrap()
     }
 
+    pub(crate) async fn request_future<P, R>(&self, method: &str, params: P) -> R
```

Comment:
Make this function asynchronous inside by using async channel from tokio when awaiting callback response